lib/mongodb.js

```diff
@@ -10,19 +10,9 @@ const options = {};
 let client;
 let clientPromise;
 
-if (process.env.NODE_ENV === "development") {
-  // In development mode, use a global variable so that the value
-  // is preserved across module reloads caused by HMR (Hot Module Replacement).
-  if (!global._mongoClientPromise) {
-    client = new MongoClient(uri, options);
-    global._mongoClientPromise = client.connect();
-  }
-  clientPromise = global._mongoClientPromise;
-} else {
-  // In production mode, it's best to not use a global variable.
-  client = new MongoClient(uri, options);
-  clientPromise = client.connect();
-}
+// no development mode. Develop in production mode
```

Comment:
just note that you can always connect to a local db or test db by changing `MONGODB_URI` in `.env.local`. If you have one 😁